src/components/404/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import NotFoundImg from '../../assets/images/404.svg';
+import styles from './styles.module.css';
+export default class NoMatch extends Component {
```

Comment:
Would you mind 🙏 changing this to a functional approach since its the approach used in all the codebase (and the latest version of React allow us to do it)
ex:
```
const NoMatch = () => (
<div>...</div>
);

export default NoMatch;
```
So far I think that if you look through all the project you don't find a single `class` 🤔 😄